astropy/io/votable/tree.py

```diff
@@ -1,37 +1,34 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 # TODO: Test FITS parsing
 
+import base64
+import codecs
+import gzip
 # STDLIB
 import io
 import re
-import gzip
-import base64
-import codecs
 import urllib.request
 import warnings
 
 # THIRD-PARTY
 import numpy as np
 from numpy import ma
 
+from astropy import __version__ as astropy_version
 # LOCAL
```

Comment:
Hmm why is this checking astropy version? Refactoring for another day lol.
```suggestion
# LOCAL
from astropy import __version__ as astropy_version
```

Reply:
👍 . Thanks for the detailed check!